client/components/happychat/composer.jsx

```diff
@@ -44,7 +44,11 @@ export const Composer = React.createClass( {
 				onMouseLeave={ this.scrollbleedUnlock }
 				>
 				<div className="happychat__message">
+					<label htmlFor="hc_message_textarea" style={{ display: "none"}}>
```

Comment:
Can't you just do `<label><textarea ...></label>`? Then you shouldn't need to add any IDs or invisible-making styles.